### ISSUE TITLE
Ability to pass custom args to helm in deployments

### ISSUE THREAD

**Issue by khaled** [devspace-cloud/devspace#1192]
**Is your feature request related to a problem?**  
Yes - I need to skip validation on a particular chart deployment via  `--disable-openapi-validation`

**Which solution do you suggest?**  

Add a `customArgs` option, e.g:

```yaml
deployments:
- name: database
  helm:
    chart:
      name: stable/mysql
    timeout: 300
    customArgs: --disable-openapi-validation
```

**Which alternative solutions exist?**  

Maybe doing a "manual" helm install via a hook? :) 

/kind feature

**Comment by FabianKramm** [comment]
@khaled thanks for opening this issue! Yes we can add that and this actually aligns well with our internal plans to drop the internal helm dependency and instead use the helm cli for v3.

**Comment by FabianKramm** [comment]
@khaled this is implemented in v5.2.0
